src/libs/ReceiptUtils.ts

```diff
@@ -36,17 +36,19 @@ function getThumbnailAndImageURIs(transaction: Transaction, receiptPath: string
 
     const hasEReceipt = transaction?.hasEReceipt;
 
-    if (hasEReceipt) {
-        return {thumbnail: null, image: ROUTES.ERECEIPT.getRoute(transaction.transactionID), transaction};
-    }
+    if (!Object.hasOwn(transaction?.pendingFields ?? {}, 'waypoints')) {
```

Comment:
I think this could be simplified
```suggestion
    if (!transaction?.pendingFields?.waypoints) {
```

Reply:
@ArekChr This line is making the type check in typescript fail. How should we change the code so the checks pass? 😄 Should we revert the line?

Reply:
I see now, let's leave it as it is

Reply:
@Tony-MK, I mean, let's leave it as it was implemented in the previous condition. :D